package.json

```diff
@@ -15,7 +15,7 @@
   "license": "MIT",
   "dependencies": {
     "querystring": "^0.2.0",
-    "strip-ansi": "^2.0.1"
+    "strip-ansi": "^3.0.0"
```

Comment:
Includes a breaking change https://github.com/chalk/strip-ansi/commit/c0a1a7db94eb593f1ba2e133cf53da8a363a31e6, bit doesn't affect this project.